### DISCUSSION TITLE
Update from 3.5.5 to 3.6.5

### DISCUSSION THREAD

**Discussion by CodeSippr** [clientIO/joint#2084] [Q&A]
### Introduction

I am currently updating my JointJS Project to 3.6.5. With this Update many Type Errors are occured. But i do not know how to fix them. I have some Classes which are extending dia.Link or dia.Element.

For Example the use of the Class "Area" gives me the following Error:

"Type 'Area' is not assignable to type 'Element<Attributes, ModelSetOptions> | Link<Attributes, ModelSetOptions>'.
  Type 'Area' is not assignable to type 'Element<Attributes, ModelSetOptions>'.
    The types returned by 'position(...)' are incompatible between these types.
      Type 'joint.g.Point' is not assignable to type 'import("/node_modules/jointjs/dist/joint").g.Point'."

Does anyone have an Idea how to fix such errors?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by jamesgeorgewilliams** [comment]
Have you updated the `@types` dependencies to latest versions? The latest version of `@types/backbone` is `1.4.15` for example.

If you are already using the latest `@types`, could you provide some examples of code that is not working in `3.6.5`?
